recipe_scrapers/goodhousekeeping.py

```diff
@@ -0,0 +1,93 @@
+# mypy: allow-untyped-defs
+
+import re
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import normalize_string
+
+
+class GoodHousekeeping(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "goodhousekeeping.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".ingredients-body h3",
+            ".ingredient-lists li",
+        )
+
+    def instructions(self):
+        directions = self.soup.find(
+            "ul",
+            class_="directions",
+        )
+
+        methods = [
+            directions.find_all("p"),
+            directions.find("li", recursive=False).find_all("li"),
+        ]
+
+        for m in methods:
+            if len(m) != 0:
+                instructions = m
+                break
+
+        return "\n".join(
+            [
+                re.sub(r"Step \d", "", normalize_string(instruction.get_text())).strip()
+                for instruction in instructions
+            ]
+        )
+
+    def ratings(self):
+        return ""
+
+    def cuisine(self):
+        return self.schema.cuisine()
```

Comment:
One of your test cases illustrates an interesting edge case for this scraper -- and probably others too which haven't handled this in the past -- sometimes the cuisine is reported as `""` (empty string).

It's a nitpick, and maybe me trying too hard to enforce consistency, but I think it'd be better to return the Python `None` value in that case.  An idea for how to do that:

```suggestion
        return self.schema.cuisine() or None
```